### PR TITLE
2. Apply Futurize stage 1 across the repository.

### PR DESCRIPTION
**Pull request by amakhaba**
This PR does the following ([Apply Futurize stage 1](https://github.com/ska-sa/katdeveloper/wiki/Investigate-a-way-to-improve-Py3-migration-feedback-loop-on-tox#4-futurize-module-maintain-py2-support)):
- Applies futurize stage 1 across the whole repository.

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] Requested at least 2 reviewers?
JIRA: [MT-738](https://skaafrica.atlassian.net/browse/MT-738)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,9 @@
       install_requires=[
           "PyTango>=9.2.2",
           "numpy",
-          "jsonschema"],
+          "jsonschema",
+          "future",
+          "futures; python_version<'3'"],
```
Comment (ajoubertza) [inline setup.py]: This is not required for stage 1.  We are not using `future` or `futures` in this PR, and `from __future__ ` is standard.

File: tango_simlib/utilities/base_parser.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```
Comment (ajoubertza) [inline tango_simlib/utilities/base_parser.py]: Please move this after the copyright block, to be consistent with the other modules - we always have the imports after the initial docstring.

File: tango_simlib/utilities/helper_module.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```
Comment (ajoubertza) [inline tango_simlib/utilities/helper_module.py]: Please move after copyright.

File: tango_simlib/utilities/testutils.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```
Comment (ajoubertza) [inline tango_simlib/utilities/testutils.py]: Move after copyright.

File: setup.py (hunk 1 above)
Comment (amakhaba) [reply setup.py]: If that is the case then it should also be removed from that Futurize stage 1 step in the migration guide, and placed in the relevant step

Comment (ajoubertza) [reply setup.py]: Sorry - I didn't notice that error in the migration guide.

File: tango_simlib/quantities.py [hunk 5]
```diff
@@ -1,3 +1,6 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```
Comment (ajoubertza) [inline tango_simlib/quantities.py]: Move these after copyright, please.

Comment (mmphego) [reply tango_simlib/quantities.py]: Cascaded PR are messed up, changes made here: https://github.com/ska-sa/tango-simlib/pull/108/files

**Comment by mmphego** [comment]
PR blocked by https://github.com/ska-sa/tango-simlib/pull/105

**Pull request merged**